server/routes/deallocate-from-activity/handlers/checkDeallocation.ts

```diff
@@ -12,7 +14,7 @@ export default class CheckDeallocationRoutes {
     const deallocationReasons = await this.activitiesService.getDeallocationReasons(user)
 
     res.render('pages/deallocate-from-activity/check-answers', {
-      deallocationDate: parseDate(deallocationDate),
+      deallocationDate: formatDate(plainToInstance(SimpleDate, deallocationDate).toRichDate(), 'yyyy-MM-dd'),
```

Comment:
This is what I originally had but had other issues when calling the API.  It was grumbling about formats.  Have you run this locally against the API?  Sorry have to ask.